app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/UIHelper.cs

```diff
@@ -130,13 +130,45 @@ public static IEnumerable<Face> CalculateFaceRectangleForRendering(IList<Detecte
 
             foreach (var face in faces)
             {
+                var Left = (int)((face.FaceRectangle.Left * scale) + uiXOffset);
+                var Top = (int)((face.FaceRectangle.Top * scale) + uiYOffset);
+
+                // FaceAngle use for roate face Rect, default is 0(no rotate).
```

Comment:
`FaceAngle used for rotating face rectangles, default value is 0 (not rotated).`

Reply:
Done.